taskcluster/rpi3-build.sh

```diff
@@ -27,11 +27,13 @@ if [ $1 = "--aot" ]; then
   BAZEL_BUILD_FLAGS="${BAZEL_BUILD_FLAGS} ${BAZEL_AOT_BUILD_FLAGS} ${AOT_MODEL_PARAMS}"
 fi;
 
+maybe_install_xldd
+
 do_bazel_build
 
 do_deepspeech_binary_build
 
-export SUPPORTED_PYTHON_VERSIONS="2.7.14:ucs4 3.4.8:ucs4"
+export SUPPORTED_PYTHON_VERSIONS="3.4.8:ucs4 3.5.3:ucs4"
```

Comment:
Moving to strech, we have 2.7 3.4 and 3.5. But PiWheels only provide prebuilt scipy / numpy for 3.4 and 3.5, nothing for 2.7.

Reply:
I just looked and saw that we don't actually depend (directly) on NumPy for the package, it's a spurious import. If we care about 2.7 support (or just simplifying the RPi3 support), we should look into using the built-in `wave` module instead of `scipy.io.wavfile`.

Reply:
(I'm OK with dropping 2.7 here)

Reply:
Yes, I agree, we should try to move away from scipy. But the problem holds for numpy, which I don't think we can move away that easily